.github/workflows/screener-run.yml

```
@@ -117,12 +159,13 @@ jobs:
 
       - name: Install dependencies
         run: yarn install --frozen-lockfile
-
       - name: Upload @fluentui/react VR test site
         uses: azure/CLI@v1
         with:
           inlineScript: |
             az storage blob upload-batch -d '$web/${{env.DEPLOYBASEPATH}}/react-screener' -s 'apps/vr-tests/dist/storybook' --overwrite
+        if: ${{ env.IS_ARTIFACT_PRESENT == 'false' }}
```

```suggestion
        if: ${{ env.IS_ARTIFACT_PRESENT == 'true' }}
```

Should it be `true` instead?